### PR TITLE
Data masking credit card number example for SQL Server

### PR DESCRIPTION
**Pull request by rodolforeitz**

### CODE REVIEW

File: masking-examples/credit-card/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```yaml
+data:
+  - CCN
+rules:
+  - reads:
+      - data:
+          - custom:${database_name}.${schema_name}.mask_ccn(CCN)
+        rows: any
+        severity: low
+```
```
Comment (juniocezar) [inline masking-examples/credit-card/README.md:93]:
```suggestion
```json
{
  "governedData": {
    "labels": [
      "CCN"
    ]
  },
  "readRules": [
    {
      "constraints": {
        "mask": {
          "function": "custom:${database_name}.${schema_name}.mask_ccn(CCN)"
        }
      }
    }
  ]
}
```
```

File: masking-examples/credit-card/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+```yaml
+data:
+  - CCN
+rules:
+  - reads:
+      - data:
+          - custom:mask_ccn(CCN)
+        rows: any
+        severity: low
+```
```
Comment (juniocezar) [inline masking-examples/credit-card/README.md:62]:
```suggestion
```json
{
  "governedData": {
    "labels": [
      "CCN"
    ]
  },
  "readRules": [
    {
      "constraints": {
        "mask": {
          "function": "mask_ccn(CCN)"
        }
      }
    }
  ]
}
```
```

**Review by juniocezar — APPROVED** [review]

File: masking-examples/credit-card/README.md (hunk 2 above)
Comment (rodolforeitz) [reply masking-examples/credit-card/README.md]: https://github.com/cyral-quickstart/quickstart-datarepo-masking/pull/19/commits/63d4d71c680a19515ba9a1b366ac46943d8476b5

File: masking-examples/credit-card/README.md (hunk 1 above)
Comment (rodolforeitz) [reply masking-examples/credit-card/README.md]: https://github.com/cyral-quickstart/quickstart-datarepo-masking/pull/19/commits/63d4d71c680a19515ba9a1b366ac46943d8476b5

**Pull request merged**